core/frontend/src/DisplayStyleState.ts

```diff
@@ -655,11 +631,7 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
 
   /* @internal */
   private _synchBackgroundMapImagery() {
-    this._backgroundMap.setBaseLayerSettings(this.settings.mapImagery.backgroundBase);
-    this._backgroundMap.setLayerSettings(this.settings.mapImagery.backgroundLayers);
-    this._backgroundDrapeMap.setBaseLayerSettings(this.settings.mapImagery.backgroundBase);
-    this._backgroundDrapeMap.setLayerSettings(this.settings.mapImagery.backgroundLayers);
-    this._overlayMap.setLayerSettings(this.settings.mapImagery.overlayLayers);
+    this.onMapImageryChanged.raiseEvent(this.settings.mapImagery);
```

Comment:
DisplayStyleSettings has an onMapImageryChanged event that is raised by synchMapImagery. Why does DisplayStyleState also need one?

Reply:
No not needed - I removed the unnecessary DisplayStyleState.onMapImageryChanged as well as DisplayStyleState.onMapSettingsChanged.